### ISSUE TITLE
NewParseFrameRate() возвращает float

### ISSUE THREAD

**Issue by Persivan** [Persivan/Eventurika-Honkai-Impact-Translation#17]
Надо проверить будут ли проблемы, если эта функция вернет 23.99999 или похожее значение (не целое число)

**Comment by Devyatyi9** [comment]
Более точный пример значения фреймрейта
Frame rate : 29.970 (30000/1001) FPS
_(29,97002997002997)_

**Comment by Devyatyi9** [comment]
Проверил через сам Scaleform VideoEncoder, оказывается проблема была изначально только в нём.
Если генерировать через гуи то может произойти такая ситуация, что программа при создании батника указывает фреймрейт **30,1** из-за чего в дальнейшем происходила ошибка.
Заменил в батнике на `-framerate=29,970`, и всё прошло успешно. (принимает только через запятую)

Это баг `ScaleformVideoEncoder.exe`

![image](https://github.com/Persivan/Eventurika-Honkai-Impact-Translation/assets/5161472/3b0cf9d0-0552-4c12-b3a8-7eeddadcde92)
![image](https://github.com/Persivan/Eventurika-Honkai-Impact-Translation/assets/5161472/49283021-a53c-40cf-ba3e-25fd046166cd)

**Comment by Devyatyi9** [comment]
Так, я теперь всё перепроверил, и вижу что на выходе 10 фпс, проверил по файлу и в логе тоже, но почему, откуда, не понятно
Это тот же файл что выше
![image](https://github.com/Persivan/Eventurika-Honkai-Impact-Translation/assets/5161472/d60b6b80-d217-4bfc-90b1-92482dcdf0c2)
На крайний случай можно захардкодить это значение, или диапазон, если вдруг попадутся такие файлы.
Ну или же просто принудительно делать округление для этого Энкодера как до этого

**Comment by Devyatyi9** [comment]
Добавил подсказку на всякий
Вдруг пригодится при конвертации из mp4 а не из m2v, где фпс уже мог быть выровнен
![image](https://github.com/Persivan/Eventurika-Honkai-Impact-Translation/assets/5161472/3f13f7b3-24ee-42f2-8f75-1b2058e63e34)
